.pre-commit-config.yaml

```diff
@@ -1,19 +1,25 @@
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v2.5.0
     hooks:
       - id: check-yaml
       - id: end-of-file-fixer
       - id: trailing-whitespace
--   repo: https://github.com/python/black
+  - repo: https://github.com/python/black
```

Comment:
Wasn't there some discussion about ignoring single- vs double- quoted strings? Should we be installing a dot file with explicit checking preferences for `black`?

Reply:
There was, in this is how you do it:

https://github.com/napari/napari/blob/master/pyproject.toml#L12

But, you cannot configure black to enforce single quotes, just to ignore them.  In the end, I wasn't sure it was worth the hassle.  Also, PrettierJS uses double quotes, so it makes things consistent.

If you'd like us to add that option, I'm happy to add the `.pyproject.toml`.